mantle/cmd/kola/testiso.go

```diff
@@ -617,13 +617,14 @@ func awaitCompletion(ctx context.Context, inst *platform.QemuInstance, outdir st
 	return time.Since(start), err
 }
 
-func printResult(test string, duration time.Duration, err error) bool {
+func printResult(test string, duration time.Duration, snoozeDate harness.DenyListObj.SnoozeDate, err error) bool {
```

Comment:
This will be reverted as the printResult func is originally supposed to return a boolean, nothing more.